source/samples/DisplayTest.cpp

```diff
@@ -50,9 +50,6 @@ const char * const sad_emoji ="\
     000,255,255,255,000\n\
     255,000,000,000,255\n";
 
-Image happy(happy_emoji);
-Image sad(sad_emoji);
-
```

Comment:
These are not even used, so they can be removed completely.
Functions in this file reuse the char arrays to generate the same Images, but don't call these directly.